tests/Support/AuthTestTrait.php

```diff
@@ -9,21 +9,23 @@ trait AuthTestTrait
 {
     use MockTrait;
 
-    public function mockOpensslRandomPseudoBytes(): void
+    public function mockOpensslRandomPseudoBytes(string $password): void
     {
         $this->mockNativeFunction('Illuminate\Auth\Passwords', [
             $this->functionCall(
                 name: 'hash_hmac',
+                arguments: ['sha256', $password, 'b"DÉV&´G"áËa\x1Ae&õš,Ü\x16½\x1A\x15‘\x07CM—ºSVxÅ\x16"'],
```

Comment:
as I can see - the last arg is the app key, let's just fill app key with the simple value (e.g. `app_key`)